pywinauto/controls/uia_controls.py

```diff
@@ -851,10 +855,19 @@ def get_item(self, row):
                     raise ValueError("Element '{0}' not found".format(row))
         elif isinstance(row, six.integer_types):
             # Get the item by a row index
-            # TODO: Can't get virtualized items that way
-            # TODO: See TODO section of item_count() method for details
-            list_items = self.children(content_only=True)
-            itm = list_items[self.__resolve_row_index(row)]
+            try:
+                com_elem = 0
```

Comment:
Is it possible that zero "handle" is passed into `UIAElementInfo.__init__`? Default root element assumes `None` otherwise it takes `ElementFromHandle(handle_or_elem)`. What `ElementFromHandle(0)` should return then? Will it raise `ComError`?

I'd suggest to use `None` and add check for `None` (raise ValueError in case of `None`).

Reply:
Good point @vasily-v-ryabov, but it shouldn't be a case here, because we always call for ```get_elem_interface(com_elem, "VirtualizedItem").Realize()``` that already handles ValueError. But I added the case to the unit tests, so we should catch the problem if code is changed.

Reply:
OKI, it seems `com_elem = 0` is always overridden by at least one iteration of the loop below since you have `non_negative_number + 1` at the end of range. But in some hypothetical situation `self.__resolve_row_index(row)` may return `-1` and `com_elem = 0` will be passed to `get_elem_interface` and it will raise `AttributeError`. Well, it looks properly handled now.